src/cf-forms/usage.md

```diff
@@ -133,15 +133,34 @@ Color variables referenced in comments are from [cf-core cf-brand-colors.less](h
 
 ### Label helper text
 
+#### Inline helper text
+
+Appears with label headings.
```

Comment:
Does this mean inline helpers are not to be used with standard labels?

Reply:
cc @nataliafitzgerald I believe you discussed this with Ans because I brought it up when he was first exploring options a couple weeks ago.

Reply:
'cause if so, what would the pattern be for indicating that a field with a basic label is required?

Reply:
From what I remember in the conversation, we don't use standard labels outside of the ones to the side of a checkbox/radio. Everything else is a label heading or a legend, which makes me realize we probably need this for legends as well...

Reply:
What Jimmy said. Helper text appears in large target checkboxes/radio buttons, but those use block helpers.

Reply:
Alright, I don't want this discussion to take this PR off the rails, but it sounds like `a-label__heading` should be the standard `a-label` and we should have a modifier for the one that sits next to a check/radio, like `a-label__inline` or something.

Reply:
(Just now realized it was merged before I commented the first time.)

Reply:
You bring up a good point @Scotchester, the general standard is `label__heading`, so why should we need to add that all the time.

Reply:
🤔 wicked smaht